src/index.html

```diff
@@ -738,6 +738,18 @@ <h3 class="facility-info__title">Loading...</h3>
             </svg></div>
         </div>
       </div>
+      <a href="#" class="facility-info__view-google-map hidden w-inline-block">
```

Comment:
Was this change made on Webflow as well? If it wasn't, there is a risk it might be overwritten by future exports.

Reply:
Yes, this code was taken from a webflow export